modules/healthcare-apis/workspace/README.md

```diff
@@ -114,7 +114,6 @@ fhirServices: [
         corsMaxAge: 600
         corsAllowCredentials: false
         location: location
-        diagnosticLogsRetentionInDays: 7
```

Comment:
This file still contains the `diagnosticLogsRetentionInDays` in the Parameter usage section for JSON. 

![image](https://github.com/Azure/ResourceModules/assets/28486158/ceeae647-a1a8-4a78-a940-00b5fbc988d9)

Reply:
removed